src/Client/AzureManaged/DurableTaskSchedulerClientExtensions.cs

```diff
@@ -85,6 +85,12 @@ static void ConfigureSchedulerOptions(
             .Configure(additionalConfig ?? (_ => { }))
             .ValidateDataAnnotations();
 
+        builder.Services.AddOptions<DurableTaskClientOptions>(builder.Name)
+            .Configure(options =>
```

Comment:
shall we chain this configure in         builder.Services.AddOptions<DurableTaskSchedulerClientOptions>(builder.Name)
            .Configure(initialConfig)
            .Configure(additionalConfig ?? (_ => { }))
            .ValidateDataAnnotations();